assets/styles/styles.css

```diff
@@ -0,0 +1,137 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-image: url("../images/images.jpeg");
+  background-size: cover;
+  background-attachment: fixed;
+  font-size: 30px;
+}
+
+.heading {
+  text-align: center;
+  width: 100%;
+  margin: auto;
+}
+
+#head {
+  font-size: 60px;
+  color: #fff;
+}
+
+#subheader {
+  font-size: 40px;
+  margin-top: 10px;
+  color: #fff;
+}
+
+#content1 {
+  width: 90%;
+  margin: auto;
+  padding: 10px;
+}
+
+form {
+  background-color: #00000080;
+  padding: 3px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 54%;
+  border-radius: 5px;
+  margin: auto;
+  margin-top: 10px;
+  padding-bottom: 60px;
+  color: white;
+}
```

Comment:
Compare your survey page with the original one you will see many changes like the fonts (family, weight, and size) and the background colors